app/web/features/communities/events/CreateEventPage.test.tsx

```diff
@@ -16,6 +16,23 @@ const createEventMock = service.events.createEvent as jest.MockedFunction<
   typeof service.events.createEvent
 >;
 
+const getAccountInfoMock = service.account.getAccountInfo as MockedService<
+  typeof service.account.getAccountInfo
+>;
+
+const accountInfo = {
+  username: "tester",
+  email: "email@couchers.org",
+  profileComplete: true,
+  phone: "+46701740605",
+  phoneVerified: true,
+  timezone: "Australia/Broken_Hill",
+  hasStrongVerification: false,
+  birthdateVerificationStatus: 1,
+  genderVerificationStatus: 3,
+  doNotEmail: false,
+};
+
 describe("Create event page", () => {
   beforeAll(() => {
     server.listen();
```

Comment:
How about adding a test to check that the popup comes up if there is no account profile?

You could mock the GetUser function to return what counts as an empty profile, then import the Dialog and check to see if it rendered.